src/server/licensing_func.c

```diff
@@ -544,6 +544,9 @@ license_one_node(pbsnode *pnode)
 				update_license_highuse();
 			} else {
 				add_to_unlicensed_node_list(pnode);
+				if (pnode->nd_attr[ND_ATR_LicenseInfo].at_val.at_long) {
+					licensing_control.licenses_total_needed += pnode->nd_attr[ND_ATR_LicenseInfo].at_val.at_long;
```

Comment:
Please use get_attr_l instead of accessing it directly.

Reply:
made the changes please have a look @arungrover